doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -241,14 +235,12 @@ Bluetooth mesh samples
 
 * :ref:`bluetooth_mesh_sensor_client` sample:
 
-  * Added:
-
-    * Ability to use buttons to send get and set messages for a sensor setting, as well as a get message for a sensor descriptor.
+  * Added the ability to use buttons to send get and set messages for a sensor setting, as well as a get message for a sensor descriptor.
```

Comment:
```suggestion
  * Added the ability to use buttons to send ``get`` and ``set`` messages for a sensor setting, as well as a ``get`` message for a sensor descriptor.
```
Not 100% sure about this, but I would prefer some highlighting for the message types to make them stand out from the text.